Alcatraz/Installers/ATZPluginInstaller.m

```diff
@@ -65,10 +66,16 @@ - (NSString *)downloadRelativePath {
 // This is a temporary support for installs in /tmp.
 - (NSString *)pathForInstalledPackage:(ATZPackage *)package {
     NSString *pluginsInstallPath = [NSHomeDirectory() stringByAppendingPathComponent:INSTALLED_PLUGINS_RELATIVE_PATH];
-    NSString *pluginInstallName = [self installNameFromPbxproj:package] ?: package.name;
+    NSString *pluginInstallName = [self installNameFromPbxproj:package] ?: [package.name stringByAppendingString:package.extension];
 
-    return [[pluginsInstallPath stringByAppendingPathComponent:pluginInstallName]
-                                       stringByAppendingString:package.extension];
+    // Packages can be installed in multiple places:
+    // ~/Library/Application Support/Developer/Shared/Xcode/Plug-ins (where most xcplugins go)
+    // ~/Library/Developer/Xcode/Plug-ins (where ideplugins go)
```

Comment:
> Avoid adding comments in the code detailing the changes, use Github issues instead

It could make sense to pull this out into a predicate method or something?